src/index.tsx

```diff
@@ -622,6 +622,8 @@ export class Slider extends PureComponent<SliderProps, SliderState> {
             thumbStyle,
             thumbTintColor,
             trackStyle,
+            minimumTrackStyle: propMinimumTrackStyle,
```

Comment:
`minimumTrackStyle` has already been defined in the code. Hence renamed the prop to `propMinimumTrackStyle`